webpages/settings/index.js

```diff
@@ -787,8 +787,7 @@ const vue = (window.vue = new Vue({
     },
     setTheme(mode) {
       chrome.storage.sync.get(["globalTheme"], function (r) {
-        let rr = true; //true = light, false = dark
-        rr = mode;
+        let rr = mode;
```

Comment:
why did you remove this logic?

Reply:
It seems to me that it will be changed no matter what, and it is just unnecessary.

Reply:
Yeah, if you look at the old logic it sets it to true then proceeded to change it to mode

Reply:
im now realizing that the logic is broken